check_process

```diff
@@ -0,0 +1,35 @@
+# See here for more information
+# https://github.com/YunoHost/package_check#syntax-check_process-file
+
+# Move this file from check_process.default to check_process when you have filled it.
+
+;; Test complet
+	; Manifest
+		domain="domain.tld"	(DOMAIN)
+		pvp=1
+		servername="packagecheck"
+		creative=0
+		damage=1
+		is_public=0	(PUBLIC|public=1|private=0)
+		password="pass"
+		port="30000"	(PORT)
+	; Checks
+		pkg_linter=1
+		setup_sub_dir=0
+		setup_root=0
+		setup_nourl=1
+		setup_private=0
+		setup_public=0
+		upgrade=1
+		backup_restore=1
+		multi_instance=0
```

Comment:
If `multi_instance` is set to `true` in the manifest, this one have to be changed to 1.